project_proposal/tex/mainpage.tex

```diff
@@ -0,0 +1,382 @@
+\section{Motivation}
+As students,
+this semester project gives the group a great learning experience as well as a good idea of how such a project would progress.
+During the project, the group will learn how to create and setup an OPC-UA client,
```

Comment:
setup and Open Platform Communications Unified Architecture, OPC-UA, client, ...